vald/healthcheck.go

```diff
@@ -101,13 +97,10 @@ func checkTofnd(ctx context.Context, _ client.Context, serverCtx *server.Context
 		panic(err)
 	}
 
-	nopLogger := server.ZeroLogWrapper{Logger: zerolog.New(io.Discard)}
```

Comment:
did this logger ever do anything?

Reply:
It logs only `nopLogger.Debug(“successful connection to fond gRPC server”)`, which is redundant because the command returns an error before logging.
```
	conn, err := tss.Connect(valdCfg.TssConfig.Host, valdCfg.TssConfig.Port, valdCfg.TssConfig.DialTimeout)
	if err != nil {
		return fmt.Errorf("failed to reach tofnd: %s", err.Error())
	}
```

Reply:
what I mean is that it's using io.Discard, so it's never writing anything

Reply:
right right